public/bellomosmap.js

```diff
@@ -1,11 +0,0 @@
-//bringing in map tile layer from website
-let myMap = L.map("mapid").setView([43.605421, -72.984491], 17);
-
-L.tileLayer("https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png", {
-  maxZoom: 19,
-  attribution:
-    '&copy; <a href="https://www.openstreetmap.org/copyright">OpenStreetMap</a> contributors',
-}).addTo(myMap);
-
-let fourthMark = L.marker([43.605421, -72.984491]).addTo(myMap);
-fourthMark.bindPopup("<h5>Bellomo's Market</h5>");
```

Comment:
rather then doing all of this specifically for each restaurant, while doing our fetch, we would also be getting back our lat, long cords and we can just plug those cords in for our line 10 `L.marker([lat, long])` and have that generate the pin. we can then also use a variable in line 11 to place the popup with the name of the restaurant